### PR TITLE
Documentation rectifications.

### PR DESCRIPTION
**Pull request by ghost**
Summary:
- Added code of conduct.
- Added contribution guide.
- Corrected README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,21 +12,10 @@ features which users would appriciate.
 AoS has a small community and it is about time we did something to make the game
 more interesting for others so they could get interested in it.
 
-## Workflow
-You want to add a packet to AoS? Great!
+## Contribute
+If you would like to contribute bug fixes, improvements, and new features please take a look at our [Contributor Guide](CONTRIBUTING.md) to see how you can participate in this open source project.
+You can also contact us via the official [discord server][discord].
 
-### Packet Proposal
```
Comment (Haxk20) [inline README.md:18]: This should 100% stay

File: README.md [hunk 2]
```diff
@@ -12,21 +12,10 @@ features which users would appriciate.
...
-File an issue and name it `PP: PacketName`. Describe what packet you wish to add,
-and how it would improve the game. It will be voted on here on github.
-If the community approves that they want this packet it will be added
-(Ofcourse unwanted or duplicated features will be denied)
-
-### Packet Add Request
```
Comment (Haxk20) [inline README.md:24]: Same here

**Review by Haxk20 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:24]: The content was moved into the CONTRIBUTING.md, do you want me to leave it in the README.md?

Comment (Haxk20) [reply README.md:24]: Ah i must have missed that

**Pull request merged**